app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,11 @@
+class CommentsController < ApplicationController
+  def index
+    @commented_post_id = params['id']
+  end
+
+  def comment
+    Comment.create(text: params['comment'], author_id: params['author_id'], post_id: params['post_id'])
```

Comment:
- [ ] Based on the requirements of this project, a comment is created on behalf of the currently logged-in user. Hence let's fix this. Remember you implemented the current_user method in your application_controller.


- [ ] As requested by the previous reviewer, you are supposed to make use of strong parameters. Kindly fix these issues.